LanguageSpecification.md

```diff
@@ -404,6 +442,24 @@ In this example, `AgeGeotiff` may refer to an External and the ages at this grid
 ### Other grid cells
 These keywords can only access information in the current grid cell. One may also query for entities through the `within` keyword which will search geospatially. 
 
+### Meta variables
+The `meta` keyword provides access to simulation metadata and context information:
+
+```
+start organism Tree
+
+  age.init
+    :if(meta.stepCount == 0 count) = sample external ObservedAges
+    :else = 0 years
+
+end organism
+```
+
+Available meta variables include:
```

Comment:
Please indicate that these are the options available by default but any values on the simulation entity can be accessed via meta.